notification-sdk/src/main/java/org/sunbird/notification/beans/EmailConfig.java

```diff
@@ -0,0 +1,63 @@
+package org.sunbird.notification.beans;
+
+public class EmailConfig {
+
+  private String fromEmail;
+  private String userName;
+  private String password;
+  private String host;
+  private String port;
+
+  public EmailConfig(String fromEmail, String userName, String password, String host, String port) {
+    super();
```

Comment:
No need of super(), since this class doesn't extending any class

Reply:
this is auto generated constructor , either u add super() explictly or not , JVM will always add it.